### PR TITLE
Accessibility changes

### PR DESCRIPTION
**Pull request by Leboroz**
We changed the alt properties of some tags and the contrast of some colors to make the page more accessible.

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -25,15 +25,15 @@
       <img
```
Comment (carlos-ssh) [inline views/index.html:25]: **Provide a title in your project**
_Better page titles:_
- Leo's portifolio
- Leo full-stack webdeveloper

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Leboroz, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: views/index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
```
Comment (ZeenatLawal) [inline views/index.html:4]: - [OPTIONAL] Your PR should have a detailed report listing all points that you have checked, and the result of your evaluation. Each point on the list needs to include the result of your evaluation: If everything was ok, make a statement about that (e.g. "Page titles - no issues found"). If you needed to change something, link to the corresponding line of code in your pull request changes.

- [OPTIONAL] The base of your PR should be the master branch, not `sass-implementation` to follow proper GitHub flow guidelines.

**Review by ZeenatLawal — APPROVED** [review]
Hi @Leboroz and @RafaelEchart, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**